docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -18,6 +18,45 @@ nameservers.
 
 {% capture steps %}
 
+## Introduction
+
+Starting from Kubernetes v1.3, DNS is a built-in service launched automatically
+using the addon manager
+[cluster add-on](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/README.md).
+
+The running Kubernetes DNS pod holds 3 containers:
```

Comment:
Humm..This section seems more like implementation details that should be developer facing. We should probably put these information on https://github.com/kubernetes/dns/blob/master/README.md instead?

Reply:
Oh just noticed this existed originally. Putting this in introduction might be confusing end users, what about keeping the "How it Works" section and move it somewhere below?

Reply:
The task is located in "administer-cluster" subdir, so we can assume that the readers are not "end" users.

Reply:
SGTM.